travis/install-tools.sh

```diff
@@ -31,6 +31,9 @@ else
     cp requirements-conda.txt $HOME
 fi
 
+# Set to conda's java
+export JAVA_HOME="$(pwd)/tools/conda-tools/default/jre"
```

Comment:
Considering the Picard wrapper locates the JRE, do we need to override `JAVA_HOME`? Could we simply `unset JAVA_HOME`?